app/assets/javascripts/Components/autotest_manager.jsx

```diff
@@ -316,6 +317,16 @@ class AutotestManager extends React.Component {
             disableActions={{rename: true}}
           />
         </fieldset>
+        <fieldset>
```

Comment:
This is good! We might decide later that we want to move this to its own page but if we do, it will be easy to just copy and paste this code.